deploy/shipping/views.sql

```diff
@@ -43,4 +43,35 @@ create or replace view shipping.reportable_condition_v1 as
 
     order by encountered desc;
 
+
+create or replace view shipping.metadata_for_augur_build_v1 as
+
+    select  sample as strain,
+            encountered_date as date,
+            'seattle' as region,
+            residence_census_tract as location,
+            'Seattle Flu Study' as authors,
+            case
+                when age < 18 then 'child'
+                else 'adult'
+            end as age_category,
+            case
+                when site_type in ('childrensHospital', 'childrensClinic', 'childrensHospital', 'clinic', 'hospital', 'retrospective') then 'clinical'
+                when site_type in ('childcare' , 'collegeCampus' , 'homelessShelter' , 'port', 'publicSpace', 'workplace') then 'community'
+            end as site_category,
```

Comment:
Just curious: what's the motivation for creating new categories of age and site here?

Reply:
It's what Trevor had requested. I believe it's used as broad filters on the tree.

Reply:
It might be nice to eventually move these into `site.details` as `category` or something.